### ISSUE TITLE
Can i run_classifiy with out pretrained model in new dataset?

### ISSUE THREAD

**Issue by kingstarcraft** [google-research/bert#195]
I use the run_classifiy.py to train mydataset ,but the recall is zero. I am not sure which reson cause that problem, mybe I have not use the pretrained model or run_pretraining.py, or may the ratio of neg to pos is too large(9000:300 per txt file).Is that any one can help me?

**Comment by HaodaY** [comment]
No, you need the pre-trained model by bert; but you need not to re-train it yourself. The only thing you need do before run_classify is writing a subclass of DataProcessor to process your data and add the processor to processor list in main function.
